linux_os/guide/system/software/integrity/software-integrity/aide/aide_check_audit_tools/ansible/shared.yml

```diff
@@ -34,9 +34,12 @@
     regexp: ^{{ item }}\s
     line: "{{ item }} {{{ aide_string() }}}"
   with_items: "{{ audit_tools }}"
+  when: "'aide' in ansible_facts.packages"
```

Comment:
The first task already ensures `aide` is installed so it is not necessary to test it in the next tasks.